modules/installing-the-odh-operator-v1.adoc

```diff
@@ -0,0 +1,37 @@
+:_module-type: PROCEDURE
+//pv2hash: 9cbc09e0-cac2-4eb3-8f30-09e6469c5164
+
+[id='installing-the-odh-operator-v1_{context}']
+= Installing the Open Data Hub Operator (v1)
```

Comment:
Similar comment to https://github.com/opendatahub-io/opendatahub-documentation/pull/55/files#r1386898401